source/uwp/SharedRenderer/lib/AdaptiveImageRenderer.cpp

```diff
@@ -414,36 +414,45 @@ namespace AdaptiveCards::Rendering::Xaml_Rendering
-                    [weakThis = this->get_weak(),
-                     weakImageSource = winrt::make_weak(image.as<winrt::SvgImageSource>()),
-                     imageUrl](auto const& operation, auto status) -> void
+                getOperation.Completed([weakThis = this->get_weak(),
+                                        weakImageSource = winrt::make_weak(image.as<winrt::SvgImageSource>()),
+                                        imageUrl](auto const& operation, auto status)
+                {
+                    if (status == winrt::AsyncStatus::Completed)
                     {
-                        auto strongThis = weakThis.get();
-                        auto strongImageSource = weakImageSource.get();
+                        winrt::HttpResponseMessage response = operation.GetResults();
 
-                        if (strongThis && strongImageSource)
+                        if (response.IsSuccessStatusCode())
                         {
-                            if (status == winrt::AsyncStatus::Completed)
-                            {
-                                auto success = strongThis->ParseXmlForHeightAndWidth(operation.GetResults(), strongImageSource);
+                            auto readOperation = response.Content().ReadAsStringAsync();
 
-                                if (success)
+                            readOperation.Completed([weakThis, weakImageSource, imageUrl](auto const& operation, auto status)
                                 {
-                                    // Now that we've parsed the height and width successfully, we can set the image source
-                                    strongThis->SetSvgUriSource(strongImageSource, imageUrl);
+                                auto strongThis = weakThis.get();
+                                auto strongImageSource = weakImageSource.get();
+
+                                if (strongThis && strongImageSource)
+                                {
+                                    if (status == winrt::AsyncStatus::Completed)
+                                    {
+                                        // Read SVG xml
+                                        winrt::XmlDocument xmlDoc;
+                                        xmlDoc.LoadXml(operation.GetResults());
+                                        auto success = strongThis->ParseXmlForHeightAndWidth(xmlDoc, strongImageSource);
+                                        if (success)
+                                        {
+                                            // Now that we've parsed the height and width successfully, we can set the image source
+                                            strongThis->SetSvgUriSource(strongImageSource, imageUrl);
+                                        }
+                                    }
                                 }
-                            }
-                            else if (status == winrt::AsyncStatus::Error)
```

Comment:
Is there a reason why we don't check for this error now?

Reply:
I accidentally added it in a previous PR. We are not doing anything with the error.